samples/web/content/apprtc/apprtc.py

```diff
@@ -333,7 +340,14 @@ def __str__(self):
 def get_memcache_key_for_room(host, room_id):
   return '%s/%s' % (host, room_id)
 
-def add_client_to_room(host, room_id, client_id, is_loopback):
+def does_room_exist(host, room_id):
```

Comment:
nit: if does_room_exist reads strangely (at least to me). Maybe just has_room(host, room_id)?